tests/testthat/test-select.R

```diff
@@ -64,6 +64,25 @@ test_that("col_select can select the id column", {
   )
 })
 
+test_that("id column not required in col_select (#416)", {
+  expect_named(
+    vroom(vroom_example("mtcars.csv"), id = "path", col_select = list(model, mpg)),
```

Comment:
```suggestion
    vroom(vroom_example("mtcars.csv"), id = "path", col_select = c(model, mpg)),
```

It feels like `c()` is more of the official tidyselect style (instead of `list()`). This applies in several places (including a few existing tests). This doesn't effect test behaviour, but seems a better (and simpler) style for us to model.

If you've come across any reason that vroom uses `list()` so much in `col_select`, let me know.

Otherwise, let's embrace `c()`. This would also need to be changed in the docs for `col_select` and the examples for `vroom()`, at the very least. You'll want to search for any other uses of `col_select`.

Relevant reference: https://tidyselect.r-lib.org/reference/language.html